### DISCUSSION TITLE
Blogger alternative

### DISCUSSION THREAD

**Discussion by Eduardo-EazyEZ** [pluja/awesome-privacy#245] [Q&A] [answered]
Please include alternatives to blogger

**Comment by Zerotask** [comment]
A possible alternative could be using [Hugo](https://gohugo.io/) which will produce a static website which you can host on a free provider like GitHub Pages, Netlify or Vercel. You have 100% control over the software, since Hugo is OSS.

**Comment by pluja** [comment, marked as answer]
added some alternatives and can be found [here](https://github.com/pluja/awesome-privacy#blogging-platforms-medium)
